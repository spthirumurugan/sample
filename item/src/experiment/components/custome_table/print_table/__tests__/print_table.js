import React from 'react';
import renderer from 'react-test-renderer';
import PrintTable from '../print_table';

jest.mock('../table_row', () => 'tr');
jest.mock('../table_header', () => 'thead');

describe('components', () => {
  describe('<PrintTable />', () => {
    it('should render', () => {
      const comp = renderer.create(
        <PrintTable viewId={'view_1'} datasetIds={['d1', 'd2']} />
      ).toJSON();
      expect(comp).toMatchSnapshot();
      // expect(toolbar.props.className).toEqual('className');
    });
  });
});
