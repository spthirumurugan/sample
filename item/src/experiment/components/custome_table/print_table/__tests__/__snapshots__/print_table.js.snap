// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <PrintTable /> should render 1`] = `
<table
  className="table table-bordered"
>
  <thead
    viewId="view_1"
  />
  <tbody>
    <tr
      datasetId="d1"
      viewId="view_1"
    />
    <tr
      datasetId="d2"
      viewId="view_1"
    />
  </tbody>
</table>
`;
